### DISCUSSION TITLE
enter the wrong address in the transfer, transfer Xcc to XCH's address.

### DISCUSSION THREAD

**Discussion by linzhiqiang0514** [Chia-Network/chia-blockchain#9073] [Support]
If I enter the wrong address in the transfer, transfer Xcc to XCH's address. This XCH address will be converted to an unknown Xcc address. Can I get it back?
I have done such a thing once on October 17.
Same address.
Is it a bug?

![image](https://user-images.githubusercontent.com/8164566/139715068-ad5dc85c-c32e-4628-acd9-ec020183c699.png)
![image](https://user-images.githubusercontent.com/8164566/139715123-bb0c0204-efe5-4a0e-84f1-3bff87c20059.png)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
